UefiPayloadPkg/SPI/SPI.inf

```diff
@@ -0,0 +1,70 @@
+#/** @file
+#
+#  Component description file for SMMSTORE module
+#
+#  Copyright (c) 2020, 9elements Agency GmbH<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#**/
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = SPI
+  FILE_GUID                      = c2c2e656-ee66-41e0-bee3-29c6f16f49c0
+  MODULE_TYPE                    = DXE_RUNTIME_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = SPIInitialize
+
+[Sources]
+  SPI.h
+  SPIgeneric.h
+  SPIgeneric.c
+  pci_type.h
+  device.h
+  kconfig.h
+  fch_spi_ctrl.h
+  fch_spi_ctrl.c
+  SPI_fvb.h
+  SPI_fvb.c
+  fch_spi_util.h
+  fch_spi_util.c
+  stopwatch.h
+  pci_devs.h
```

Comment:
these files must go away
- pci_devs.h
- pci_ops.h
- pci_ops.c

Reply:
pci_devs.h is now deleted. I'm still working on the rest of them

Reply:
pci_ops.h, pci_ops.c are now deleted too